seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java

```diff
@@ -83,33 +86,103 @@ private JsonToRowConverter createNotNullConverter(SeaTunnelDataType<?> type) {
             case ROW:
                 return createRowConverter((SeaTunnelRowType) type);
             case NULL:
-                return jsonNode -> null;
+                return new JsonToRowConverter() {
```

Comment:
Add comment about why not use lambda.

Reply:
https://stackoverflow.com/questions/28946655/invalid-lambda-deserialization-when-lambda-used-for-interface-implementation @Hisoka-X

Reply:
It is for developers which want change the code.